backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -529,21 +387,175 @@ class SchedulingService implements ISchedulingService {
+                    <br />
+                    The following details have been updated:
+                    </p>`
+                  : `Thank you for scheduling a donation to your local community fridge.
+                    <br />
+                    <br />
+                    Here is a summary of your upcoming donation:`
+              }
+              
+     
+              <div style="display: flex; flex-direction: row; align-content: flex-start; gap: 40px;">
+                <div style="padding-right: 40px">
+                  <h2 style="font-weight: 600; font-size: 18px; line-height: 28px; color: #171717;">
+                      Proposed drop-off time
+                  </h2>
+                  <p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">
+                      ${startDayString}
+                      <br />
+                      ${startTimeString} - ${endTimeString}
+                      <br />
+                      ${frequencyString}
+                  </p>
+                </div>
+              <div>
+                  <h2 style="font-weight: 600; font-size: 18px; line-height: 28px; color: #171717;">
+                    Donation information
+                  </h2>
+                  <p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">
+                      ${schedule.size} - ${donationSizeDescriptions.get(
+        schedule.size ?? "",
+      )}
+                  <br/>
+                  ${schedule.categories.join(", ")}
+              </p>
+          </div>
+     </div>
+     
+     
+     <h2 style="font-weight: 600; font-size: 18px; line-height: 28px; color: #171717;">
+         Volunteer information
+     </h2>
+     <p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">
+         ${
+           schedule.volunteer_needed
+             ? "Volunteer required"
+             : "Volunteer not required"
+         }
+     <br/>
+     ${schedule.is_pickup ? "Pickup required" : ""}
+     <br/>
+     ${schedule.is_pickup ? schedule.pickup_location : ""}
+     <br/>
+     ${schedule.notes ? `Additional Notes: ${schedule.notes}` : ""}
+     <br/>
+     </p>
+        
+     ${
+       updated
+         ? `<p style="font-weight: 400; font - size: 16px; line - height: 24px; color: #171717; ">If this is an error, please contact the CFKW admin team.</p>`
+         : ""
+     }
+     
+     <p style="margin-top: 50px; font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">Sincerely,</p>
+     <p style="font-weight: 400; font-size: 16px; line-height: 24px; color: #171717;">Community Fridge KW</p>   
+     </body>
+     </html>
+      `;
+
+      this.emailService.sendEmail(
+        currDonor.email,
+        updated
+          ? `DONATION EDITED - Donation on ${startDayString}`
+          : "Your Donation Information",
+        emailBody,
+      );
```

Comment:
doesn't this no longer include the changes for sending the correct email copy to **admin** for new donations?
https://github.com/uwblueprint/community-fridge-kw/pull/123

Reply:
Looks like my branch was outdated and didn't have that PR, updated now